benchmark/util/priority-queue.js

```diff
@@ -7,7 +7,7 @@ const bench = common.createBenchmark(main, {
 }, { flags: ['--expose-internals'] });
 
 function main({ n, type }) {
-  const PriorityQueue = require('internal/priority_queue');
+  const PriorityQueue = require('internal/timers/priority_queue');
```

Comment:
Perhaps we should move this benchmark file to the `timers` directory now?

Reply:
If you do that (and I think it's a fine idea), be sure to run `test/benchmark/test-benchmark-timers` to make sure the move doesn't break that test file. (If it does break it, it should be easy to fix, but you may have to know something about how the benchmark tests work. Happy to help with that, although I'm guessing you'll figure it out if you aren't familiar with the benchmark tests already.)

Reply:
^^^^ You'll want to run that test locally because they don't run in node-test-pull-request. (But they do run in node-daily-master so if it breaks, we'll find out, but not until after the code lands!)

Reply:
Yep👍.

And I've run `python tools/test.py -J --mode=release benchmark/test-benchmark-timers` and this benchmark test passed, so I think it should be fine :)

Reply:
By the way you can use https://ci.nodejs.org/job/node-test-commit-custom-suites-freestyle/ to run this in the CI for this PR (use `pummel` in `CI_JS_SUITES`)

Reply:
Cool, thank you.